packages/twenty-front/src/modules/ui/input/components/__stories__/Select.stories.tsx

```diff
@@ -1,12 +1,16 @@
-import { useState } from 'react';
+import { ReactNode, useState } from 'react';
 import { Meta, StoryObj } from '@storybook/react';
 import { userEvent, within } from '@storybook/test';
 
+import { IconComponent } from '@/ui/display/icon/types/IconComponent';
 import { ComponentDecorator } from '~/testing/decorators/ComponentDecorator';
 
 import { Select, SelectProps } from '../Select';
 
-type RenderProps = SelectProps<string | number | null>;
```

Comment:
Could you improve the storybook to add a variation with avatar/icon ?

Reply:
Added the variation :D